oxidation/libparsec/crates/client_types/src/wasm32/local_device_file.rs

```diff
@@ -0,0 +1,76 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use std::convert::Infallible;
+use std::path::Path;
+
+use crate::{local_device_file::*, StrPath};
+
+impl DeviceFile {
+    fn load(data: &[u8]) -> Result<Self, rmp_serde::decode::Error> {
+        rmp_serde::from_slice(data)
+    }
+}
+
+/// TODO: Remove me once wasm doesn't need to mock
+pub fn test_gen_default_devices() {
```

Comment:
Is this really useful? Can't we just ask devs to add a device to their local storage? This way we can control exactly what device, the backend it's linked to, it's auth method, if there's a human_handle, length of email and name, ... and it doesn't pollute the code with things that we'll forget to remove.
I think it's better to set the environment in which the code is running rather than let the code define its own environment.